src/ServiceControl/CompositeViews/Endpoints/GetKnownEndpointsApi.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace ServiceControl.CompositeViews.Endpoints
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Nancy;
+    using Nancy.Extensions;
+    using ServiceControl.CompositeViews.Messages;
+    using ServiceControl.Monitoring;
+
+    public class GetKnownEndpointsApi : ScatterGatherApi<NoInput, List<KnownEndpointsView>>
+    {
+        public EndpointInstanceMonitoring EndpointInstanceMonitoring { get; set; }
+
+        public override Task<QueryResult<List<KnownEndpointsView>>> LocalQuery(Request request, NoInput input, string instanceId)
+        {
+            var result = EndpointInstanceMonitoring.GetKnownEndpoints();
+            return Task.FromResult(Results(result, instanceId));
```

Comment:
I think this is wrong. The call to `Results` without a stats object returns `QueryStats.Zero` instance which has an empty etag and 0 count on the results. In reality, we probably don't want this call to have an etag or total count.

Reply:
I have no idea what to do here. Are you saying we should "spoof" `QueryStats` @mikeminutillo ?

Reply:
@mikeminutillo would PR #1097 address your concern?

Reply:
It does